### DISCUSSION TITLE
Move table of contents ToC to bottom left corner

### DISCUSSION THREAD

**Discussion by Kani3D** [TriliumNext/Trilium#3874] [Q&A]
Hello,

is it possible to move the table of contents to the bottom left corner?
![image](https://user-images.githubusercontent.com/105086349/233952903-cb13dfaa-5842-4c55-993b-172c769209c7.png)

I found this but it should be outdated, since it's built in now, right?
https://github.com/zadam/trilium/discussions/2799

**Comment by zadam** [comment]
Hi, this is not supported in the built-in feature. I don't know about the plugin.
